Fallout Stuff

NigglyWiggly opened the pull request:
Bunch of Fallout related stuff.
New version of bottle Nuka Cola, not the homemade bar drink (though
they're the same reagent), Nuka Cola Quantum, a vending machine that
sells them both, Vault 13 and General Vault-Tec Uniforms, and a 'Booze
Bunker' bar sign.

After this, I'm gonna start working on the Pip Boy 2000 PDA alternative,
which'll probably just be random space loot

NigglyWiggly commented:
Oh wait, that quantum reagent thing probably doesn't actually do anything, lemme fix that real quick

Nienhaus commented:
Here. http://puu.sh/iiYcU/4b13289b2b.dmi I also have a couch and pool table.

Kurfursten commented:
In your testing, how irradiated does someone get from one bottle of quantum?

NigglyWiggly commented:
Just a bit more than with uranium

Kurfursten commented:
How much radiation does 30u uranium put into someone?

NigglyWiggly commented:
Probably around 3u of mutagen worth

FudgePucker commented:
Wait uranium isn't even that effective of a toxin?
It's only obtainable from the roid, why is it weaker than something out of the chemical desp?

ComicIronic commented:
Because uranium's main use isn't as a poison, it's a material.

NigglyWiggly commented:
Probably gonna need assistance with the Pip boy thing, so I'm gonna make that a whole different PR

9600bauds commented:
Uranium is actually not that radioactive.

ComicIronic commented:
While you're doing the editing, could you fix the typo on nuka_cola that stops it applying life effects?

NigglyWiggly commented:
Sure thing

ririchiyo commented:
Please fix up the relative pathing in some of the files.

9600bauds commented:
New()? Near damn ALL food items are like that.
Take a peak at snacks.dm.

NigglyWiggly commented:
So is there actually a problem with the Travis thing, or is Riri just giving me more reasons to tell him he's a butt?

clusterfack commented on code/modules/reagents/reagent_containers/food/drinks.dm:
```diff
@@ -666,6 +666,28 @@
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/quantum
+	name = "Nuka Cola Quantum"
+	desc = "Take the leap... enjoy a Quantum!"
+	icon_state = "quantum"
```
Need an extra line here

NigglyWiggly commented:
So it only started fucking up the travis stuff after I changed the pathing, would anyone other than Riri have an objection to if I just reverted back to before then?

PJB3005 commented:
/me goes insane about relative pathing

NigglyWiggly commented:
Relative pathing schmelative pathing merge it anyway bby

9600bauds commented:
What's so bad anyways? Every food item has a new() since it's the glorified reagent list. That single harmless bit of relative pathing keeps one food item per leftmost tabulation and makes the file actually easier to read.
Not like anyone will ctrl+f "soda_cans/nuka/new()" anyways.
